.github/workflows/rolling_build.yml

```
@@ -24,24 +24,41 @@ jobs:
       uses: carlosperate/download-file-action@v1.0.3
       with:
         # URL of the file to download
-        file-url:  ${{ secrets.CRM_SKILLS_FLAG_URL }}
+        file-url: ${{ secrets.CRM_SKILLS_FLAG_URL }}
 
     # Check if we need to run a build on production
     - name: Run the Build if CRM has updated
       run: |
+        # Check if the file contains "TRUE" string
         if grep -q TRUE "CRM_SkillsUpdatedFlag.txt"; then
+            # Setup variables for the next steps
+            echo "should_trigger_deploy=TRUE" >> $GITHUB_ENV
+            echo "branch_name=main" >> $GITHUB_ENV
+            
+            # Avoid triggering the build again by overriding the file with FALSE
             echo FALSE > CRM_SkillsUpdatedFlag.txt
+
             echo "Running Build - CRM Updated"
-            export AZURE_DEVOPS_EXT_PAT=${{ secrets.AZURE_DEVOPS_TOKEN }}
-            chmod +x .workflow/build-production.sh
-            bash ./.workflow/build-production.sh ${{ github.token }}
+            echo "Dispatching main-trigger-people-deploy workflow"
         else
+            echo "should_trigger_deploy=FALSE" >> $GITHUB_ENV
             echo "Skipping Build - CRM Data Ok"
-        fi
       
-    # Update the CRM Flag so the build is triggered on next change  
+    # Trigger the other workflow if the CRM data was updated
+    - name: Build and deploy People
+      if: env.should_trigger_deploy == 'TRUE'
+      uses: benc-uk/workflow-dispatch@v1
+      with:
+        workflow: main-trigger-people-deploy.yml
+        repo: SSWConsulting/SSW.People.Profiles
+        ref: ${{ env.branch_name }}
```

I expected this to be
```suggestion
        workflow: build-deploy-production.yml
        repo: SSWConsulting/SSW.People
        ref: ${{ steps.find_branch.outputs.branch_name }}
```
Instead of triggering another workflow to trigger another workflow

---

This intentionally trigger `main-trigger-people-deploy.yml` so that we only have one workflow that needs to generate GitHub token in order to trigger the workflow in a different repo.

Otherwise, I would need to write this instead:

```yaml
      - name: Find latest branch
        id: find_branch
        run: echo "branch_name=$(git ls-remote --heads origin | grep -oe 'refs/heads/release/.*' | sort -V | tail -n 1 | sed 's|refs/heads/||')" >> $GITHUB_OUTPUT          

        # SSW GitHub App - Trigger Workflows
      - name: Generate a token
        id: generate-token
        uses: actions/create-github-app-token@v1
        with:
          app-id: ${{ vars.APP_ID }}
          private-key: ${{ secrets.APP_PRIVATE_KEY }}
          owner: SSWConsulting
          repositories: SSW.People

      - name: Build and deploy People
        uses: benc-uk/workflow-dispatch@v1
        with:
          workflow: build-deploy-production.yml
          repo: SSWConsulting/SSW.People
          ref: ${{ steps.find_branch.outputs.branch_name }}
          token: "${{ steps.generate-token.outputs.token }}"
```